UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/ExternalBodyHealthEffect.cs

```diff
@@ -52,15 +52,15 @@ public class ExternalBodyHealthEffect : MetabolismReaction
 						foreach (var Effect in Effects)
 						{
 							bodyPart.RelatedPart.TakeDamage(null,
-								Effect.EffectPerOne * ReagentMetabolismMultiplier * -OverdoseDamageMultiplier,
+								Effect.EffectPerOne * -OverdoseDamageMultiplier * ReagentMetabolismMultiplier,
```

Comment:
isn't this exactly the same?

Reply:
yeah, was fiddling around with the calculations, came back to the same thing